client/my-sites/stats/stats-email-detail/index.jsx

```diff
@@ -111,12 +111,14 @@ class StatsEmailDetail extends Component {
 			insights: this.props.translate( 'Insights' ),
 			store: this.props.translate( 'Store' ),
 			ads: this.props.translate( 'Ads' ),
+			subscribers: this.props.translate( 'Subscribers' ),
 		};
 		const possibleBackLinks = {
 			traffic: '/stats/day/',
 			insights: '/stats/insights/',
 			store: '/stats/store/',
 			ads: '/stats/ads/',
+			subscribers: '/stats/subscribers',
```

Comment:
this needs a trailing `/` at the end as it's not currently directing to the right place without it. So it should be `/stats/subscribers/`

Reply:
Good catch! Updated!